application/analyst/services/state_updater.py

```
@@ -29,6 +29,27 @@ def _normalize_text(value: Any) -> str:
     return str(value or "").strip().lower()
 
 
+def _safe_int(value: Any, default: int) -> int:
+    """尽力从任意值中解析整数，失败时回退默认值。"""
```

Ruff (0.15.10)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)